Exercise 1 BFile/myFile-rt/src/com/tridiumuniversity/myFile/BFile.java

```diff
@@ -0,0 +1,220 @@
+package com.tridiumuniversity.myFile;
+
+
+import javax.baja.nre.annotations.*;
+import javax.baja.sys.*;
+
+@NiagaraType
+@NiagaraProperty(
+  name = "fileName",
+  type = "String",
+  defaultValue = "Shubham",
+  flags = Flags.READONLY | Flags.SUMMARY
+)
+@NiagaraProperty(
+  name = "path",
+  type = "BString",
+  defaultValue = "BString.DEFAULT",
+  flags = Flags.READONLY
+)
+@NiagaraProperty(
+  name = "size",
+  type = "int",
+  defaultValue = "2",
+  flags = Flags.SUMMARY
+)
+@NiagaraProperty(
+  name = "createdDate",
+  type = "BAbsTime",
+  defaultValue = "BAbsTime.DEFAULT",
+  flags = Flags.READONLY
+)
+@NiagaraProperty(
+  name = "modifiedDate",
+  type = "BAbsTime",
+  defaultValue = "BAbsTime.DEFAULT",
+  flags = Flags.TRANSIENT
```

Comment:
This is fine, although for a real class modeling a file, you'd probably want to preserve the `modifiedDate` even when the station is shut down.